tools/UpdatePaths/Scripts/73415_simple_fugu_to_basic.txt

```diff
@@ -0,0 +1,3 @@
+# replace simple animal fugu with basic animal
+
+/mob/living/simple_animal/hostile/asteroid/fugu/@SUBTYPES : /mob/living/basic/wumborian_fugu/@SUBTYPES{@OLD}
```

Comment:
it's probably fine to leave it as subtypes but in the future, updatepaths should be written for expected changes (unless you know of a place that actually subtyped this fuckin' mob)